aws/plugin.go

```diff
@@ -254,6 +254,7 @@ func Plugin(ctx context.Context) *plugin.Plugin {
 			"aws_serverlessapplicationrepository_application":              tableAwsServerlessApplicationRepositoryApplication(ctx),
 			"aws_servicequotas_default_service_quota":                      tableAwsServiceQuotasDefaultServiceQuota(ctx),
 			"aws_servicequotas_service_quota":                              tableAwsServiceQuotasServiceQuota(ctx),
+			"aws_servicequotas_service_quota_change_request":               tableAwsServiceQuotasServiceQuotaChangeRequest(ctx),
```

Comment:
@johnsmyth Looking at this table name again, I wonder if it should be `aws_servicequotas_service_quota_increase_request`? The [ListRequestedServiceQuotaChangeHistory API](https://docs.aws.amazon.com/servicequotas/2019-06-24/apireference/API_ListRequestedServiceQuotaChangeHistory.html) does use the word `change` over `increase`, but a lot of other AWS documentation, and some API calls like [RequestServiceQuotaIncrease](https://docs.aws.amazon.com/servicequotas/2019-06-24/apireference/API_RequestServiceQuotaIncrease.html) use the word `increase` over `change`.

Thoughts?

Reply:
@cbruno10 I dont have a preference. I dont have a lot of exposure to this API, but the docs look like they use both terms.